api/api/spec/spec.yaml

```diff
@@ -1182,192 +1368,254 @@ components:
+          - syn_sent
+          - syn_recv
+          - fin_wait1
+          - fin_wait2
+          - time_wait
+          - close
+          - close_wait
+          - last_ack
+          - listening
+          - closing
+          - delete_tcp
+          - unknown
         tx_queue:
+          description: Packets pending to be transmitted.
           type: integer
           minimum: 0
         rx_queue:
+          description: Packets at the receiver queue.
           type: integer
           minimum: 0
         inode:
+          description: Port inode.
           type: integer
         protocol:
           type: string
+          enum:
+          - tcp
+          - udp
+          - tcp6
+          - udp6
         agent_id:
+          $ref: '#/components/schemas/AgentID'
+        pid:
+          description: PID owner of the opened port.
+          type: integer
+        process:
+          description: Name of the PID.
           type: string
-          minLength: 3
 
     SyscollectorProcess:
       type: object
       properties:
         scan:
-          type: object
-          properties:
-            id:
-              type: integer
-            time:
-              type: string
-              format: date-time
+          $ref: '#/components/schemas/ScanIdTime'
         fgroup:
+          description: Filesystem group name.
           type: string
         state:
+          description: Process state.
           type: string
         priority:
+          description: Kernel scheduling priority.
           type: integer
         suser:
+          description: Saved-set user.
           type: string
         ppid:
+          description: Process parent PID.
           type: integer
           minimum: 0
         vm_size:
+          description: Total VM size (KB)
           type: integer
           minimum: 0
         session:
+          description: Process session.
           type: integer
         pid:
+          description: Process PID.
           type: string
         nlwp:
+          description: Number of light weight processes.
           type: integer
         name:
+          description: Process name.
           type: string
         euser:
+          description: Effective user.
           type: string
         sgroup:
+          description: Saved-set group
           type: string
         size:
+          description: Process size in bytes.
           type: integer
           minimum: 0
         stime:
+          description: Time spent executing system code.
           type: integer
           minimum: 0
         nice:
+          description: Nice value of the process
           type: integer
         argvs:
+          description: Process received arguments.
           type: string
         share:
+          description: Shared memory.
           type: integer
           minimum: 0
         processor:
+          description: Processor number which is running the process.
           type: integer
+          minimum: 0
```

Comment:
Add format: int32

Reply:
Done in 266632160480aff4ab91873668d658ef85dcec2b